ci/docker/templates/s3-coldstorage/nuxeo.defaults

```diff
@@ -0,0 +1,5 @@
+## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
+#GLACIER
+s3-coldstorage.target=.
+nuxeo.glacier.bucket=
+nuxeo.glacier.bucket_prefix=
```

Comment:
can you please point me to the place where and how the `s3-glacier-blob-provider-config.xml.nxftl` will be processed

Reply:
I can't directly answer the question but I can redirect you to the [Nuxeo Document](https://doc.nuxeo.com/nxdoc/next/configuration-templates/). or I'm simply not understand your question